src/test/resources/org/openstreetmap/atlas/geography/atlas/dynamic/DynamicAtlasPreemptiveLoadTest.osm

```diff
@@ -0,0 +1,41 @@
+<?xml version='1.0' encoding='UTF-8'?>
```

Comment:
Is the .osm file an intended addition?

Reply:
Yes, I used JOSM to create it, and this way the test data is easy to open in JOSM.

Reply:
Yeah, I saw that. Just this is a first for the Atlas project. We might consider adding a "Test best practices" section - explaining that we have TestAtlas files, .osm files, .txt files and .atlas.gz in some rare cases and when it's best to use each one.